LatestTagDocs/RevenueCat.doccarchive/data/documentation/revenuecat/v4_api_migration_guide.json

```diff
@@ -1 +1 @@
com.revenuecat.Purchases/documentation/RevenueCat/Package/storeProduct":{"role":"symbol","title":"storeProduct","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"storeProduct"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"StoreProduct","preciseIdentifier":"c:@M@RevenueCat@objc(cs)RCStoreProduct"}],"abstract":[{"type":"text","text":"The underlying "},{"type":"reference","isActive":true,"identifier":"doc:\/\/com.revenuecat.Purchases\/documentation\/RevenueCat\/Package\/storeProduct"}],"identifier":"doc:\/\/com.revenuecat.Purchases\/documentation\/RevenueCat\/Package\/storeProduct","kind":"symbol","type":"topic","url":"\/documentation\/revenuecat\/package\/storeproduct"},"doc://com.revenuecat.Purchases/documentation/RevenueCat/Offering":{"role":"symbol","title":"Offering","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"Offering"}],"abstract":[{"type":"text","text":"An offering is a collection of "},{"type":"reference","isActive":true,"identifier":"doc:\/\/com.revenuecat.Purchases\/documentation\/RevenueCat\/Package"},{"type":"text","text":"s, and they let you control which products"},{"type":"text","text":" "},{"type":"text","text":"are shown to users without requiring an app update."}],"identifier":"doc:\/\/com.revenuecat.Purchases\/documentation\/RevenueCat\/Offering","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Offering"}],"url":"\/documentation\/revenuecat\/offering"},"doc://com.revenuecat.Purchases/documentation/RevenueCat/StoreTransaction/productIdentifier":{"role":"symbol","title":"productIdentifier","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"productIdentifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"The product identifier."}],"identifier":"doc:\/\/com.revenuecat.Purchases\/documentation\/RevenueCat\/StoreTransaction\/productIdentifier","kind":"symbol","type":"topic","url":"\/documentation\/revenuecat\/storetransaction\/productidentifier"},"doc://com.revenuecat.Purchases/documentation/RevenueCat/DeferredPromotionalPurchaseBlock":{"role":"symbol","title":"DeferredPromotionalPurchaseBlock","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"DeferredPromotionalPurchaseBlock"}],"abstract":[{"type":"text","text":"Deferred block for "},{"type":"reference","isActive":true,"identifier":"doc:\/\/com.revenuecat.Purchases\/documentation\/RevenueCat\/Purchases\/shouldPurchasePromoProduct(_:defermentBlock:)"}],"identifier":"doc:\/\/com.revenuecat.Purchases\/documentation\/RevenueCat\/DeferredPromotionalPurchaseBlock","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"DeferredPromotionalPurchaseBlock"}],"url":"\/documentation\/revenuecat\/deferredpromotionalpurchaseblock"}}}
```

Comment:
not sure what happened in this file

Reply:
This file had a reference to `integrationTests`. It was used to point folks to the workaround for the pre-compiled header you need to add for objective-c projects that expose our public types to a Swift target